index.html

```diff
@@ -4,15 +4,46 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<!--End-->
     <link rel="stylesheet" href="style.css">
     <title>Setup and Mobile first</title>
 </head>
 <body>
-    <div class="container">
-        <div class="box">B 1</div>
-        <div class="box one">B 2</div>
-        <div class="box">B 3</div>
-        <div class="box">B 4</div>
+<header>
+    <div class="top-logo">
+        <a href="#">Abbas</a>
     </div>
+    <nav class="Navbar">
+        <div class="menu-icon">
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+            <div class="menu-vector"></div>
+        </div>
+    </nav>
+</header>
+    <main>
+        <div class="informative-top">
+            <div class="top-heading">
+                <h1>Hello, My name is Abbas Glad to see you!</h1>
```

Comment:
- [ ] In the Figma Design template, this text is bolder than what you have in your project. Kindly look into this and fix it.

**Your Project**
![image](https://user-images.githubusercontent.com/84156714/219337930-a8e65c95-a995-4070-b9a5-4b403d45b942.png)

**Figma Design**
![image](https://user-images.githubusercontent.com/84156714/219338101-faa17278-305c-4a85-8bdb-5146c47f55e3.png)